common.blocks/i-bem/__collection/i-bem__collection.js

```diff
@@ -164,7 +164,7 @@ var BemCollection = inherit(/** @lends BemCollection.prototype */{
     /**
      * Returns a new collection comprised of collection on which it is called joined with
      * the collection(s) and/or array(s) and/or entity(es) provided as arguments.
-     * @param {Collection|Array|BemEntity} [args...]
+     * @param {?...(Collection|Array|BemEntity)} args
```

Comment:
BTW, `{?Collection}` means that parameter can be a Collection or **null**, e.g. "Nullable type". Optional parameter must be declared as `{Collection} [args]` (or `{Collection=}`, which is weird).